### DISCUSSION TITLE
What do we use "Issues" for? Does it help to label the issues?

### DISCUSSION THREAD

**Discussion by rgkirch** [phronmophobic/membrane#45] [General]
Would it help anyone if I open an issue and tag it as like "code refactor" or "bug".
I opened an [issue](https://github.com/phronmophobic/membrane/issues/44) that really isn't a bug. I just thought it was worth tracking. Although, maybe that should just be a pr instead...

**Comment by phronmophobic** [comment]
Not sure how much labeling helps for issues. Having a good title makes a big difference though.

**Comment by rgkirch** [comment]
Yea, if you already read every issue then I can see that tagging doesn't add any value.
Feel free to rename any issues I make with better titles.
